editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -1217,10 +1217,11 @@ Error EditorSceneImporterGLTF::_parse_meshes(GLTFState &state) {
 			}
 		}
 
+		mesh.blend_weights.resize(mesh.mesh->get_blend_shape_count());
+
 		if (d.has("weights")) {
 			const Array &weights = d["weights"];
 			ERR_FAIL_COND_V(mesh.mesh->get_blend_shape_count() != weights.size(), ERR_PARSE_ERROR);
```

Comment:
```suggestion
			ERR_FAIL_COND_V(mesh.blend_weights != weights.size(), ERR_PARSE_ERROR);
```

Reply:
There's an error in this code, but I get the idea.